### PR TITLE
correct sep outside oe text error

### PR DESCRIPTION
**Pull request by j1joey**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [x] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

# What is the ticket # detailing the issue?

Ticket: [ME-185178380](https://www.pivotaltracker.com/n/projects/2640060/stories/185178380)

# A brief description of the changes

Current behavior: OE date was not displaying when signing up SEP

New behavior: OE date was displaying when signing up SEP

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

### CODE REVIEW

File: components/benefit_markets/db/seedfiles/translations/en/insured.rb [hunk 1]
```diff
@@ -252,7 +252,7 @@
   "en.insured.families.final_sep_experienced_any_events" => "Have you or anyone in your household recently experienced any of the following events?",
   "en.insured.families.final_sep_none_of_the_situations_listed" => "None of the situations listed above apply.",
   ":en.insured.families.open_enrollment" => "Open Enrollment",
-  ":en.insured.families.final_sep_outside_open_enrollment_body" => "Open enrollment starts on %{next_ivl_open_enrollment_date}. To enroll before open enrollment, you must qualify for a special enrollment period. If none of the circumstances listed apply to you, you will not be able to enroll until %{next_ivl_open_enrollment_period}",
+  ":en.insured.families.final_sep_outside_open_enrollment_body" => "Open enrollment starts on November 1. To enroll before Open Enrollment, you must qualify for a Special Enrollment Period.",
```
Comment (jacobkagon) [inline components/benefit_markets/db/seedfiles/translations/en/insured.rb]: @j1joey few things. 
1. Do we want to make this change for both DC and ME? If only ME, then we should only change the ME translation.
2. Let's avoid hard coding November 1st. There's probably a Resource Registry setting somewhere that has November 1st. Let's try to use that.

**Review by jacobkagon — CHANGES_REQUESTED** [review]
see comments

**Comment by jayreddy519** [comment]
Branch is RED @j1joey

**Review by jacobkagon — APPROVED** [review]

**Review by jacobkagon — CHANGES_REQUESTED** [review]
@j1joey small change, but this is the requested text:

 "Open enrollment starts on November 1. To enroll before Open Enrollment, you must qualify for a Special Enrollment Period."

You need to capitalize Open Enrollment and Special Enrollment Period

**Review by jacobkagon — DISMISSED** [review]
@j1joey small change, but this is the requested text:

 "Open enrollment starts on November 1. To enroll before Open Enrollment, you must qualify for a Special Enrollment Period."

You need to capitalize Open Enrollment and Special Enrollment Period

**Review by jayreddy519 — APPROVED** [review]

**Review by jacobkagon — APPROVED** [review]

**Pull request closed without merging**